src/pip/_internal/commands/wheel.py

```diff
@@ -177,6 +186,19 @@ def run(self, options, args):
                         raise CommandError(
                             "Failed to build one or more wheels"
                         )
+                    if wb.path_to_wheelnames is not None:
+                        entries_to_save = wb.wheel_filenames[::]
+                        for req in requirement_set.requirements.values():
+                            if req.link.filename.endswith('whl'):
+                                entries_to_save.append(req.link.filename)
+
+                        with open(wb.path_to_wheelnames, 'w') as file:
+                            file.write(
+                                '\n'.join(
```

Comment:
I remember this was `os.linesep` before. What was the reason to replace it with `\n`?

Reply:
Thanks.